PasswordGenerator.py

```diff
@@ -0,0 +1,41 @@
+# The main purpose of this is to get secure password to defend against Bruteforce attacks.
+
+
+import string
+import random
+import re
+
+if __name__ == '__main__':
```

Comment:
What kind of algorithm is that?

Reply:
The algorithm should be a function that returns a password (str) and does not print().  See CONTRIBUTING.md